### DISCUSSION TITLE
Changing Notification Icon

### DISCUSSION THREAD

**Discussion by connor-devuk** [NativePHP/laravel#315] [Q&A] [answered]
Hey guys so im aware you can change the app icon by uploading your own Icon.png in the vendor files however I was wondering if you guys knew of a way to also change the icon that appears on mac with notifications I've tried digging around the vendor files but couldn't find one to replace

**Comment by simonhamp** [comment, marked as answer]
@connor-devuk it's not possible currently, but there's an [Idea open for this already](https://github.com/orgs/NativePHP/discussions/143)

Please cast your vote and join the discussion there
